aries-site/src/layouts/content/Subsection.js

```diff
@@ -4,7 +4,13 @@ import { Anchor, Box } from 'grommet';
 import { Link as LinkIcon } from 'grommet-icons';
 import { Subheading } from '../../components';
 
-export const Subsection = ({ children, name, ...rest }) => {
+const GAP_SIZE = {
+  1: 'medium',
+  2: 'small',
+  3: undefined,
```

Comment:
I might be missing something, but Gap should except more than two values.

Reply:
This gap formatting affects line 37. It helps to ensure that the spacing between the heading and the first element that follows it has the appropriate spacing. The comment on line 32 aims to explain why an h3 should not have any added gap.